src/Cameras/babylon.arcRotateCamera.ts

```diff
@@ -315,11 +315,11 @@ module BABYLON {
                 this.inertialAlphaOffset *= this.inertia;
                 this.inertialBetaOffset *= this.inertia;
                 this.inertialRadiusOffset *= this.inertia;
-                if (Math.abs(this.inertialAlphaOffset) < Epsilon)
+                if (Math.abs(this.inertialAlphaOffset) < this.speed * Epsilon)

```

Comment:
It is a kind of a breaking change as speed == 2 by default. But I guess this is okay epsilon  x 2 is still epsilon:)